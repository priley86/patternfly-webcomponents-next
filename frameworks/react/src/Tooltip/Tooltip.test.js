/* eslint-env jest */
/* 
  JS Dom Web Component Polyfill Workaround for Jest:
  https://github.com/tmpvar/jsdom/issues/1030

  Will be resolved in: 
  https://github.com/tmpvar/jsdom/pull/1872
*/
import 'document-register-element'

import React from 'react'
import renderer from 'react-test-renderer'

import { Tooltip } from '../index.js'

test('Tooltip renders default properly', () => {
  const component = renderer.create(
    <div>
      <Tooltip id="tooltip-right" placement="right" targetSelector="#btn-right">
        On the right.
      </Tooltip>
      <button
        id="btn-right"
        className="btn btn-danger"
        aria-describedby="tooltip-right"
      >
        Right
      </button>
    </div>
  )

  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
