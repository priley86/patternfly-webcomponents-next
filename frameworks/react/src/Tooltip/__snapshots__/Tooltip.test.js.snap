// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tooltip renders default properly 1`] = `
<div>
  <pf-tooltip
    container-selector={undefined}
    id="tooltip-right"
    placement="right"
    target-selector="#btn-right"
    targetSelector="#btn-right"
  >
    On the right.
  </pf-tooltip>
  <button
    aria-describedby="tooltip-right"
    className="btn btn-danger"
    id="btn-right"
  >
    Right
  </button>
</div>
`;
